src/utils/config.ts

```diff
@@ -22,6 +22,12 @@ const configParsers = {
 
 		return key;
 	},
+	LANG(key: string) {
+		parseAssert('LANG', key, 'Cannot be empty');
+		parseAssert('LANG', /^["']?[a-zA-Z\s-]+["']?$/.test(key), 'Must contain letters and dashes only');
```

Comment:
I think quotes should not be allowed. And what language do we need to support with `\s`?

```suggestion
		parseAssert('LANG', /^[a-z-]+$/i.test(key), 'Must only be letters and dashes');
```

Reply:
I was thinking about allowing languages like: `"british english"` (that's the reason to allow quotes and spaces), but that can be resolved by using the ISO notation, like `en-gb`.

Quotes and white spaces are not necessary, so I remove them from the validation :+1:

Reply:
Gotcha, sounds good.

FYI `\s` matches tabs and new-lines too: https://www.w3schools.com/jsref/jsref_regexp_whitespace.asp